### PR TITLE
Document meaning of log levels

### PR DESCRIPTION
**Pull request by hamelin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,34 @@ To install on Linux:
...
+## Log level meaning
+
+1. **CRITICAL** -- An unrecoverable error condition has been encountered.
+   - Expect things to go boom.
+   - In any case, discard all telemetry generated beyond this message.
+   - This occurrence should be investigated so as to determine whether it has
+     been caused by the simulated model or by ITsim routines. In the latter
+     case, submit a bug report; any data collected from the simulation that
+     you can share would help.
+1. **ERROR** -- An error condition has been encountered, and this condition
```
Comment (mattcraddock) [inline README.md]: Are the labels all supposed to be 1's?

File: README.md [hunk 2]
```diff
@@ -14,3 +14,34 @@ To install on Linux:
...
+   will either be fixed, recovered or tolerated.
+   - The simulation should keep running.
+   - Telemetry beyond this message is likely good, but should be treated as
+     suspicious.
+   - This occurrence should be investigated so as to determine whether it has
+     been caused by the simulated model or by ITsim routines. In the latter
+     case, submit a bug report; any data collected from the simulation that
+     you can share would help.
+1. **WARNING** -- An anomalous condition has been encountered, but this is
```
Comment (mattcraddock) [inline README.md]: What makes an error different from a warning in this case, if they're both anomalies that don't invalidate the data and call for the same response?

File: README.md (hunk 1 above)
Comment (hamelin) [reply README.md]: When Markdown is rendered, all these 1's are processed into the coherent numerals of an ordered list.

File: README.md [hunk 3]
```diff
@@ -14,3 +14,34 @@ To install on Linux:
...
+   - Expect things to go boom.
+   - In any case, discard all telemetry generated beyond this message.
+   - This occurrence should be investigated so as to determine whether it has
+     been caused by the simulated model or by ITsim routines. In the latter
+     case, submit a bug report; any data collected from the simulation that
+     you can share would help.
+1. **ERROR** -- An error condition has been encountered, and this condition
+   will either be fixed, recovered or tolerated.
+   - The simulation should keep running.
+   - Telemetry beyond this message is likely good, but should be treated as
```
[the remark this answers is not in the document]
Comment (hamelin) [reply README.md]: Excellent question. I will come back with a better proposition.

File: README.md (hunk 2 above)
Comment (hamelin) [reply README.md]: Great question again; better proposition incoming.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hamelin) [reply README.md]: So, ERROR-level logs do not infirm the validity of the output anymore. Should an error condition break telemetry, it should indeed be treated as CRITICAL.

File: README.md (hunk 2 above)
Comment (hamelin) [reply README.md]: The latest version identifies the distinction between an ERROR and a WARNING. The latter is not an error, but an unlikely code path being taken, against common expectations. The former is a problem that, in order to recover from, some computation time must be expended. It must preferrably be fixed. In both cases, telemetry output is not broken.

**Comment by hamelin** [comment]
Ready for another go @mattcraddock

**Comment by plbeaubien-eai** [comment]
I guess I'm a day late here: the final version makes perfect sense to me, providing clear guidance for both the developers and users of the simulator!

**Pull request merged**